init

```diff
@@ -76,4 +99,6 @@ mount --move /sys /newroot/system/sys || rescue_shell
 mount --move /run /newroot/system/run || rescue_shell
 
 msg "Hi!"
-exec switch_root /newroot/system /usr/bin/init || rescue_shell
+exec env -i \
+	"TERM=linux" \
+        switch_root /newroot/system /usr/bin/init || rescue_shell
```

Comment:
TERM=linux sometimes fix terminal bugs (We need global live environment)

Reply:
nice